Diagram/AutoLayoutSettings.cs

```diff
@@ -76,6 +76,20 @@ public Algorithm Algorithm
                 Run();
             }
         }
+        private bool preservePorts;
+        [Parameter]
+        public bool PreservePorts

```

Comment:
This property doesn't seem to require an explicit backing field.

Reply:
Done